### ISSUE TITLE
Change in PMA version causes access denied

### ISSUE THREAD

**Issue by ronaldmulero** [lando/lando#906]
Bug Report
==========

lando v3.0.0-beta.39 on OSX 10.13.4

As of the morning of April 18, 2018, the following "latest" phpmyadmin setting fails : 
```yml
...
  pma:
    type: phpmyadmin
...
```
Steps to reproduce:
```
1. lando rebuild -y
2. Navigate to phpmyadmin in web browser
3. Access denied: Cannot connect: invalid settings.
```
A temporary solution is to use phpmyadmin version 4.6 instead, like so:
```yml
...
  pma:
    type: phpmyadmin:4.6
...
```
```
1. lando rebuild -y
2. Navigate to phpmyadmin in web browser
3. Works.
```
Maybe it's a bug in latest phpmyadmin:
https://microbadger.com/images/phpmyadmin/phpmyadmin

Could better documentation have prevented this issue? if so what should we change?
No

Does this bug prevent you from using lando?
No

**Comment by Shurrir** [comment]
Same issue here on two different computers.
Confirmed that switching to phpmyadmin:4.6 solves this

Seems pretty bad that these versions are changed on a lando start?

**Comment by ramseyp** [comment]
I've had this happen to a few people on my team, with these versions of lando:

v3.0.0-beta.31
v3.0.0-beta.39
v3.0.0-beta.40

Our lando files were configured like: 

```
...
  pma:
    type: phpmyadmin
...
```

For each person seeing the problem, declaring the version by adding `:4.6` to the "type" line solved the problem. 

Seems odd that it would a lando bug as opposed to one in the latest phpmyadmin.

**Comment by pirog** [comment]
closing this. fixed as part of #908
